### PR TITLE
ci: add test cases for kubectl exec command

### PR DESCRIPTION
**Pull request by liubin**
Add cases for test `kubectl exec`'s options, input, and return values.

Fixes: #2765

Signed-off-by: bin liu <bin@hyper.sh>

### CODE REVIEW

File: integration/kubernetes/runtimeclass_workloads/busybox-pod.yaml [hunk 1]
```diff
@@ -14,13 +14,19 @@ spec:
   containers:
   - name: first-test-container
     image: busybox
+    env:
+    - name: CONTAINER_NAME
+      value: "first-test-container"
     command:
         - sleep
-        - "30"
+        - "60"
```
Comment (jodh-intel) [inline integration/kubernetes/runtimeclass_workloads/busybox-pod.yaml:22]: Why do we need these changes?

Comment (liubin) [reply integration/kubernetes/runtimeclass_workloads/busybox-pod.yaml:22]: I'm not sure, but be afraid of if the container will exit during the test are running. In that case kubectl exec maybe get "unable to upgrade connection: container not found" error

**Review by teawater — APPROVED** [review]

**Pull request merged**